core/src/main/scala/org/bitcoins/core/script/ScriptProgram.scala

```diff
@@ -143,18 +154,26 @@ case class ExecutionInProgressScriptProgram(
     originalScript: List[ScriptToken],
     altStack: List[ScriptToken],
     flags: Seq[ScriptFlag],
-    lastCodeSeparator: Option[Int])
+    lastCodeSeparator: Option[Int],
+    trueCount: Int,
```

Comment:
Can we just make this a type please? `ConditionalCounter` or something like that?

Reply:
Also then we can link to the PR that sipa implements this on, which is extremely useful reading